DevExtreme CLI: add React Next.js info (#7295)
DevExpress/devextreme-documentation: concepts/50 React Components/50 Application Template/05 Generate a New Application.md

> DevExtreme React Application Template can leverage the following two frameworks: Vite or Next.js. Specify the `--app-type` argument to select the framework. If you omit the argument, the command prompts you to make that choice.

Reviewer: ```suggestion
DevExtreme React Application Template can leverage the following two frameworks: Vite or Next.js. To select the framework, specify the `--app-type` argument. If you omit the argument, the command prompts you to choose.
```

> You can create a new DevExtreme React Application Template with Vite or Next.js. Specify the `devextreme-cli new` command's `--app-type` argument to select an app type. If you leave the argument unspecified, the command prompts you to choose an app type.

Reviewer 2: I think "with" is kind of vague in technical contexts with lots of terms that might be not clear to the reader. You can use "with" in the sense of "with the help of" or "together with":

- Create an application with Vite or Next.js. 
- Create an application with DevExtreme UI Widgets. 

Not a big deal... And simpler sentences are okay. But I'd like to change it, because I wanted to simplify the second sentence anyway:

```suggestion
DevExtreme React Application Template can leverage the following two frameworks: Vite or Next.js. Specify the `--app-type` argument to select the framework. If you omit the argument, the command prompts you to make that choice.
```

    // ===== generate a react app with Vite =====
    npx devextreme-cli new react-app app-name --app-type=vite
    // ===== generate a react app with Next.js =====
    npx devextreme-cli new react-app app-name --app-type=nextjs

> [note] For additional information about `devextreme-cli new react-app`, refer to the following help topic: [DevExtreme CLI - Create a New Application](/Documentation/Guide/Common/DevExtreme_CLI/#DevExtreme_Application/Create_a_New_Application).

Reviewer: ```suggestion
[note] For more information about `devextreme-cli new react-app`, refer to the following help topic: [DevExtreme CLI - Create a New Application](/Documentation/Guide/Common/DevExtreme_CLI/#DevExtreme_Application/Create_a_New_Application).
```

> [note] For more information on the `devextreme-cli new react-app` command, refer to the [DevExtreme CLI - Create a New Application](/Documentation/Guide/Common/DevExtreme_CLI/#DevExtreme_Application/Create_a_New_Application) help topic.

Reviewer 2: If you have a lengthy link (or file name, or even a long property name), it's best to put that link/name at the very end, so that the reader doesn't need to read through it while keeping in mind the sentence flow. 

 ```suggestion
[note] For additional information about this command (`devextreme-cli new react-app`), refer to the following help topic: [DevExtreme CLI - Create a New Application](/Documentation/Guide/Common/DevExtreme_CLI/#DevExtreme_Application/Create_a_New_Application).
```

Author: I think we can further simplify the first sentence of the note here by omitting "this command":
```suggestion
[note] For additional information about `devextreme-cli new react-app`, refer to the following help topic: [DevExtreme CLI - Create a New Application](/Documentation/Guide/Common/DevExtreme_CLI/#DevExtreme_Application/Create_a_New_Application).
```

> Execute the following commands to run the generated application:

Reviewer 2: ```suggestion
Execute the following commands to run the application:
```
